README.md

```diff
@@ -13,26 +13,27 @@ Once installed ,there appears a Deepl back end module with a settings tab.
 ## Requirements
 - TYPO3 8.5 to 8.7.99 (V1.0.0 - 1.0.1)
 - TYPO3 9.5.1 to 9.5.99 (V1.0.2 onwards)
+- TYPO3 9.5.1 to 10.4.99 (V1.1.2 onwards)
```

Comment:
Do we really want to keep updating the requirements or just refer to extensions.typo3.org / Packagist?

Reply:
I am used to the system requirements also simply being in the README. That's why I thought an update was ok here. What do you say?

Reply:
Sure. That was just a suggestion. (As extension author you already have to update the version constraints in several places, e.g. ext_emconf.php, composer.json etc. I personally find it a nuisance and in practice often this gets neglected.)

Actually it's a little beyond the scope of your PR anyway - so just ignore my comment for now. Thanks and thanks for fixing the README.